test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNet/AspNetWebApi2Tests.cs

```diff
@@ -46,58 +66,94 @@ public AspNetWebApi2TestsCallTargetClassic(IisFixture iisFixture, ITestOutputHel
     public class AspNetWebApi2TestsCallTargetIntegrated : AspNetWebApi2Tests
     {
         public AspNetWebApi2TestsCallTargetIntegrated(IisFixture iisFixture, ITestOutputHelper output)
-            : base(iisFixture, output, enableCallTarget: true, classicMode: false)
+            : base(iisFixture, output, enableCallTarget: true, classicMode: false, enableFeatureFlag: false)
+        {
+        }
+    }
+
+    [Collection("IisTests")]
+    public class AspNetWebApi2TestsCallTargetClassicWithFeatureFlag : AspNetWebApi2Tests
+    {
+        public AspNetWebApi2TestsCallTargetClassicWithFeatureFlag(IisFixture iisFixture, ITestOutputHelper output)
+            : base(iisFixture, output, enableCallTarget: true, classicMode: true, enableFeatureFlag: true)
         {
         }
     }
 
+    [Collection("IisTests")]
+    public class AspNetWebApi2TestsCallTargetIntegratedWithFeatureFlag : AspNetWebApi2Tests
+    {
+        public AspNetWebApi2TestsCallTargetIntegratedWithFeatureFlag(IisFixture iisFixture, ITestOutputHelper output)
+            : base(iisFixture, output, enableCallTarget: true, classicMode: false, enableFeatureFlag: true)
+        {
+        }
+    }
+
+    [UsesVerify]
     public abstract class AspNetWebApi2Tests : TestHelper, IClassFixture<IisFixture>
     {
         private readonly IisFixture _iisFixture;
+        private readonly string _testName;
 
-        public AspNetWebApi2Tests(IisFixture iisFixture, ITestOutputHelper output, bool enableCallTarget, bool classicMode)
+        public AspNetWebApi2Tests(IisFixture iisFixture, ITestOutputHelper output, bool enableCallTarget, bool classicMode, bool enableFeatureFlag)
```

Comment:
Consider renaming parameter `enableFeatureFlag` to something more obvious (_which_ feature flag?). Something like `enableRouteTemplateResourceNames` (but shorter?). Thanks!

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/1454/commits/20bec17c173d003e9d861571b0461ff5746bc898